app/controllers/drivers_controller.rb

```diff
@@ -0,0 +1,70 @@
+require 'money'
+class DriversController < ApplicationController
+
+  def index
+    @drivers = Driver.all
+  end
+
+  def show
+    @driver = Driver.find_by(id: params[:id])
+    if @driver.nil?
+      head :not_found
+      return
+    end
+  end
+
+
+  def new
+    @driver = Driver.new
+  end
+
+  def create
+    @driver = Driver.new(driver_params)
+    if @driver.save
+      redirect_to drivers_path(@driver.id)
```

Comment:
By the way, this is wrong-- this should be `driver_path(@driver.id)`! You'll notice that after you create a driver, the URL is `localhost:3000/drivers.777` instead of `/`, there's ` .`!